src/less/overwrites/_forms.less

```diff
@@ -23,7 +23,7 @@ label{
 }
 
 textarea{
-  resize: none;
+  // resize: none;
```

Comment:
Please read the ticket accordingly: It said "Enable vertical resizing" not both ways. 
resize: vertical; is the proper way.